src/category_theory/is_connected.lean

```diff
@@ -36,14 +36,24 @@ We give some equivalent definitions:
 
 We also prove the result that the functor given by `(X × -)` preserves any
 connected limit. That is, any limit of shape `J` where `J` is a connected
-category is preserved by the functor `(X × -)`.
+category is preserved by the functor `(X × -)`. This appears in `category_theory.limits.connected`.
 -/
 
 universes v₁ v₂ u₁ u₂
 
+noncomputable theory
+
 open category_theory.category
+
 namespace category_theory
 
+/--
+A possibly empty category for which every functor to the discrete category is constant.
```

Comment:
```suggestion
A possibly empty category for which every functor to a discrete category is constant.
```
How about calling this `is_preconnected`? That's what the topology library does.

Reply:
Sure!